opencog/haskell/OpenCog/AtomSpace/Api.hs

```diff
@@ -0,0 +1,55 @@
+{-# LANGUAGE ForeignFunctionInterface #-}
+
+module OpenCog.AtomSpace.Api (
+      asAddNode
+    , runOnNewAtomSpace
+    , AtomSpace
+    ) where
+
+-- Note that I don't export the AtomSpace data constructors nor the asDelete/asNew functions.
+
+import Foreign
+import Foreign.C.Types
+import Foreign.C.String
+import Control.Exception
+import OpenCog.AtomSpace.Types
+import Control.Monad.IO.Class
+import Control.Monad.Trans.Reader
+import Data.Functor
+
```

Comment:
@MarcosPividori I would recommend that you use restrictions on the imports, such that
```haskell
import MyMod (myFun1, myFun2, etc)
```
This minimizes the chances of having ambiguities, but even in the absence thereof, it helps guys like me who are not very familiar with the plethora of Haskell modules to see right away which function belongs to which module.